docs/developer-guide/joins/join-streams-and-tables.md

```diff
@@ -40,6 +40,9 @@ CREATE STREAM pageviews_enriched AS
   EMIT CHANGES;
 ```
 
+> Note: Currently, the join expression must be a simple comparison.
```

Comment:
```suggestion
!!! note
    The join expression must be a simple comparison. Non-equi joins,
```

Reply:
We use `>` for "note" boxes below, too.
```
>Note: Only stream-stream joins are windowed.
```

Should we update all of them?

Reply:
@mjsax If you feel like it, but it's not a big deal.

Reply:
i think even this " must be a simple comparison" could be tightened up a little - to me a "simple comparison" is not quite the same as "an equality check" - could maybe include things like !=, >=, etc

Reply:
Doesn't the second sentence not cover it: `Non-equi joins ... are not supported.` ?

Reply:
absolutely it does! just suggesting that the first sentence could also be more specific. not a big deal.